agreement/player_permutation_test.go

```diff
@@ -811,16 +811,16 @@ func TestPlayerPermutation(t *testing.T) {
 }
 
 func playerPermutationCheck(t *testing.T, enableDynamicFilterTimeout bool) {
-	// create a protocol version where dynamic filter is enabled
-	version, _, configCleanup := createDynamicFilterConfig()
+	// create a protocol disabledFilterTimeoutVersion where dynamic filter is disabled
+	disabledFilterTimeoutVersion, _, configCleanup := overrideConfigWithDynamicFilterParam(false)
```

Comment:
I'm confused why `false` is hardcoded.  Isn't the idea to be able to check both ways, based on the argument?  I think I would give `enabledFilterTimeoutVersion` to this call, and then remove the check on line 822. Instead, always set the `inMsg.Proto`.